### PR TITLE
feat(sampling): make sampling toggle a slider instead

### PR DESCRIPTION
**Pull request by yakkomajuri**
I kind of doubt we'll end up going with a slider approach in the end (but product thinking for this feature is still going on!), but at least for now having a slider is great for testing the feature out and seeing how the results change, so thought it'd be super useful during the development process.

Also adds a link to the sampling docs so this is in theory ready for Alpha testers (need to update the docs though).

Screenshot 2023-02-27 at 15 53 04

### CODE REVIEW

**Review by pauldambra — APPROVED** [review]
Would need to lemonise if we keep it, but agree it's a great way to test the feature 👨‍🍳 👌

File: frontend/src/scenes/insights/EditorFilters/SamplingFilter.tsx [hunk 1]
```diff
@@ -28,27 +28,35 @@ export function SamplingFilter({ filters: editorFilters, insightProps }: EditorF
...
                 <div className="SamplingFilter">
-                    <LemonSwitch
-                        checked={!!filters.sampling_factor}
-                        label={
-                            <>
-                                <span>Show sampled results</span>
-                            </>
-                        }
-                        onChange={(newChecked) => {
+                    <Slider
```
[the remark this answers is not in the document]
Comment (yakkomajuri) [reply frontend/src/scenes/insights/EditorFilters/SamplingFilter.tsx:38]: I see nothing on the codepen, but I'm sure this component has some `displayValue` prop, let me see

[the remark this answers is not in the document]
Comment (yakkomajuri) [reply frontend/src/scenes/insights/EditorFilters/SamplingFilter.tsx:38]: Hmm only keeping the tooltip open, which is not great. I'll get this in and then do the logic work to get this working just like the experiments slider

[the remark this answers is not in the document]
Comment (pauldambra) [reply frontend/src/scenes/insights/EditorFilters/SamplingFilter.tsx:38]: Ha none of the links work 🤦 

It's the [Slider with InputNumber](https://ant.design/components/slider#components-slider-demo-input-number)

**Pull request merged**